homeassistant/components/goodwe/sensor.py

```diff
@@ -190,3 +197,30 @@ def available(self) -> bool:
         return cast(GoodweSensorEntityDescription, self.entity_description).available(
             self
         )
+
+    async def async_reset(self, now):
```

Comment:
This should be marked as a callback
```suggestion
    @callback
    def async_reset(self, now):
```